ValheimPlus/Hud.cs

```diff
@@ -62,7 +62,16 @@ private static void Postfix(Skills __instance, Skills.SkillType skillType, float
 					__instance.m_player.Message(MessageHud.MessageType.TopLeft, skill.m_info.m_skill + " [" + Helper.tFloat(skill.m_accumulator, 2) + "/" + Helper.tFloat(skill.GetNextLevelRequirement(), 2) + "] (" + Helper.tFloat(percent, 0) + "%)", 0, skill.m_info.m_icon);
 				}
 			}
+		}
 
+		[HarmonyPatch(typeof(Hud), "DamageFlash")]
+		public static class RemoveDamageFlash
+		{
+			private static void Prefix(Hud __instance)
```

Comment:
Needs to be PostFix or have a return false added

Reply:
I've made the change now to use Postfix instead. Please check [this commit](https://github.com/valheimPlus/ValheimPlus/pull/141/commits/b9849cdcbfd6305d8c9bacf8978c6bd04628278a)